source/userguide/getting_started.rst

```diff
@@ -153,7 +153,6 @@ macOS:
 
    curl https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.7.0.tar.gz -o elasticsearch-6.7.0.tar.gz
```

Comment:
@arushir : should we recommend ES7 now instead?

Reply:
I think we can stick to 6.7 here.

Reply:
yes!